ocs_ci/templates/logging-deployment/clusterlogging_operator/cl-og.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  generateName: openshift-logging-
```

Comment:
- at the end by intention?

Reply:
Yes, "openshift-logging-"  when left like that, It generates name like "openshift-logging-xbhg736" something like this